toolkit/imageconfigs/packagelists/base-image-packages.yaml

```diff
@@ -0,0 +1,27 @@
+packages:
```

Comment:
`base-image-packages` and `core-packages-image` are kind of confusing names. I think you are copying the existing names from the json files. But I wonder if we should take the opportunity to improve the names somewhat.